src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -53,7 +53,10 @@ func DeserializeState(x Bytes) State {
 ////////////////////////////////////////////////////////////////////////////////
 
 // TODO: placeholder epoch value -- this will be set later
-const MAX_PROVE_COMMIT_SECTOR_EPOCH = block.ChainEpoch(3)
+const MAX_PROVE_COMMIT_SECTOR_PERIOD = block.ChainEpoch(3)
+const MAX_SURPRISE_POST_RESPONSE_PERIOD = block.ChainEpoch(4)
```

Comment:
- this should probably be defined elsewhere -- it's used in a lot of parts of the codebase.
- Make sure to update or remove `POST_PROVING_PERIOD` in `post_generator.go`